homeassistant/components/roon/media_player.py

```diff
@@ -233,12 +233,13 @@ def _parse_now_playing(self, player_data):
         }
         now_playing_data = None
 
+        media_position = convert(player_data.get("seek_position"), int, 0)
+
         try:
             now_playing_data = player_data["now_playing"]
             media_title = now_playing_data["three_line"]["line1"]
             media_artist = now_playing_data["three_line"]["line2"]
             media_album_name = now_playing_data["three_line"]["line3"]
-            media_position = convert(now_playing_data["seek_position"], int, 0)
             media_duration = convert(now_playing_data.get("length"), int, 0)
             image_id = now_playing_data.get("image_key")
```

Comment:
Should these also be moved out?

Reply:
No - they work fine - and those fields are not duplicated outside of `now_playing`.